mk_mingw.mak

```diff
@@ -89,7 +86,6 @@ V_WINDRES   = $(V_WINDRES_$(V))
 V_WINDRES_0 = @echo [WINDRES] $@;
 V_WINDRES_1 =
 
-
```

Comment:
Very very very minor thing. The line deletion may not be needed.

Reply:
I've reverted the change.